packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -120,23 +123,26 @@ const validateSdp = (sdp) => {
     if (hostCandidate.length === 0) {
       LoggerProxy.logger.error('PeerConnectionManager:index#validateSdp --> iceCandidate: no IPV4 candidate present');
       validSdp = validSdp && false;
+      errorMessage = 'iceCandidate: no IPV4 candidate present';
 
       return;
     }
 
     if (SDP.BAD_MEDIA_PORTS.includes(mediaLine.port)) {
       LoggerProxy.logger.error('PeerConnectionManager:index#validateSdp --> iceCandidate: Found invalid port number for the ice candidate');
       validSdp = validSdp && false;
+      errorMessage = 'iceCandidate: Found invalid port number for the ice candidate';
 
       return;
     }
     if (!mediaLine.icePwd || !mediaLine.iceUfrag) {
       LoggerProxy.logger.error('PeerConnectionManager:index#validateSdp --> iceCandidate: ice ufrag and password not found');
       validSdp = validSdp && false;
+      errorMessage = 'iceCandidate: ice ufrag and password not found';
     }
   });
 
-  return validSdp;
+  return {validSdp, errorMessage};
```

Comment:
You could throw an exception instead of returning an `error` message with the valid SDP.

Reply:
I agree with @mwegman, we should just use JS flow control and throw an exception.
This way you don't have to do error handling everywhere else and just let the exception bubble up.